doc/extdev/nodes.rst

```diff
@@ -3,6 +3,8 @@
 Doctree node classes added by Sphinx
 ====================================
 
+.. note:: This page does not list the complete set of nodes available in sphinx. Please also see the `nodes provided by docutils.nodes <http://epydoc.sourceforge.net/docutils/private/docutils.nodes-module.html>`_
```

Comment:
It seems this page has been no longer maintained. This is a document for docutils-0.3.0. It's better to use https://docutils.sourceforge.io/docs/ref/doctree.html instead. This is well maintained upstream's page.

Reply:
Oops. I already commented as same...